test/validate-groups-configuration.test.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, it } from 'vitest'
+
+import { validateGroupsConfiguration } from '../utils/validate-groups-configuration'
+
+/**
+ * It is currently not possible to test rules that throw errors (https://github.com/eslint/eslint/issues/13434),
```

Comment:
Let's try to stick to the 80 character line length limit rule as stated in the prettier config?
I suggest breaking the comment down into a few lines.

Reply:
Fixed